### PR TITLE
doc: add french "point remarquable"

### PR DESCRIPTION
**Pull request by bougue-pe**
link it to operational point

### CODE REVIEW

File: content/docs/railway-wiki/glossary/_index.fr.md [hunk 1]
```diff
@@ -124,4 +126,4 @@ description: Glossaire thématique d'OSRD et du ferroviaire
 - **VS** : Voie de service
 - **VU** : Voie unique
 - **VUT** : Voie unique temporaire
-- **VUTP** : Voie unique temporaire à caractère permanent
\ No newline at end of file
+- **VUTP** : Voie unique temporaire à caractère permanent
```
Comment (shenriotpro) [inline content/docs/railway-wiki/glossary/_index.fr.md:129]: I can't seem to notice the change here, was there a missing newline or something?

File: content/docs/railway-wiki/glossary/_index.fr.md [hunk 2]
```diff
@@ -92,6 +92,8 @@ description: Glossaire thématique d'OSRD et du ferroviaire
 - **PL** : Pleine ligne
 - **PLD** : Point limite de domaine : point où une transition entre ETCS et un autre système de signalisation a lieu
 - **PN** : Passage à niveau
+- **PO** : Point opérationnel : collection de points représentant un lieu d'intérêt, voir [exemple](../../explanation/models/data-models-full-example/#points-opérationnels-operationalpoints
```
Comment (shenriotpro) [inline content/docs/railway-wiki/glossary/_index.fr.md:95]: It looks like we're missing a parenthesis.

File: content/docs/explanation/models/data-models-full-example/index.fr.md [hunk 3]
```diff
@@ -350,9 +350,11 @@ Pour plus de détails sur le modèle, voir la [page dédiée](../neutral_section
 
 #### Points opérationnels (`OperationalPoints`)
 
-Les `OperationalPoints` sont des collections de points (`OperationalPointParts`) d'intérêt.
+Le *point operationnel* est aussi connu sous le nom de *Point Remarquable* (PR).
+Un `OperationalPoint` est une **collection** de points (`OperationalPointParts`) d'intérêt.
 
-Par exemple, il peut être pratique de stocker l'emplacement des plateformes en tant que parties et de les regrouper par station dans des points opérationnels.
+Par exemple, il peut être pratique (repère de conduite) de stocker l'emplacement des plateformes en tant que parties et de les regrouper par station dans des points opérationnels.
+De la même manière  , un pont au-dessus des voies sera **un** OperationalPoint, mais il comportera **plusieurs** OperationPointParts, une à l'intersection de chaque voies.
```
Comment (shenriotpro) [inline content/docs/explanation/models/data-models-full-example/index.fr.md:357]: nit: it looks like there is a whitespace before a comma

Comment (shenriotpro) [reply content/docs/explanation/models/data-models-full-example/index.fr.md:357]: I also believe "chaque voie" should be singular.

**Review by shenriotpro — CHANGES_REQUESTED** [review]
Thanks, I noticed a few minor issues

**Comment by bougue-pe** [comment]
Thanks @shenriotpro for the review: update done, I was too sketchy in the train :pray:

**Review by shenriotpro — APPROVED** [review]

**Review by axrolld — APPROVED** [review]

**Pull request merged**